content/en/blog/2022/how-to-get-started-with-opentelemetry-go.md

```diff
@@ -0,0 +1,435 @@
+---
+title: Instrument your Go web application with openTelemetry
+linkTitle: Get Started with OpenTelemetry & Go
+date: 2022-06-27
+author: Team Aspecto
+canonical_url: https://www.aspecto.io/blog/opentelemetry-go-getting-started/
+---
+
+# How to Get Started with OpenTelemetry Go
+
+In this guide, you will learn hands-on how to create and visualize traces with OpenTelemetry Go without prior knowledge.
+
+We will start with creating a simple to-do app that uses Mongo and the Gin framework. Then, we will send tracing data to Jaeger Tracing and to Aspecto for visualization. You can find all the relevant files in this [Github repository](https://github.com/aspecto-io/opentelemetry-examples/tree/master/go).
+
+!["OpenTelemetry Go The Mandalorian"](https://www.aspecto.io/wp-content/uploads/2022/06/OpenTelemetry-Go-The-Mandalorian-2048x1406.png)
```

Comment:
I have my issues with this picture, since The Mandalorian is a trademark, and there's probably also some copyrights on the images.

Reply:
@svrnm , Hi! Do you mind to clarify your concerns regarding that illustration? Our designer made it so there aren't any copyright issues in that aspect.

Reply:
I have no legal expertise to make a clear decision if this picture (and its name & description) are OK the way they are presented here or who is liable if they are not (Linux Foundation, Aspecto, your designer?), so I rather prefer not having it. @cartermp, @chalin, @austinlparker: thoughts?

Reply:
I will ask here at the CNCF and get back to you.

Reply:
@caniszczyk says that it's ok if @NavehMevorach wants to use this image.

Reply:
thanks for clarification, @chalin & @caniszczyk!

Reply:
Thanks for bring up the issue.